src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommandProviders.cs

```diff
@@ -39,11 +39,12 @@ public class RestoreCommandProviders
         private static IReadOnlyList<IVulnerabilityInformationProvider> CreateVulnerabilityInfoProviders(IReadOnlyList<IRemoteDependencyProvider> remoteProviders)
         {
             List<IVulnerabilityInformationProvider> providers = new(remoteProviders.Count);
+            using var sourceCacheContext = new SourceCacheContext();
```

Comment:
To match naming in the `SourceRepositoryDependencyProvider` and other places..
```suggestion
            using var cacheContext = new SourceCacheContext();
```

Reply:
There's heavy usage of both tbh. 

![image](https://github.com/NuGet/NuGet.Client/assets/2878341/b251e8e8-71b1-4d2e-8d91-942dc071fad5)

Reply:
FWIW I like sourceCacheContext better.

Reply:
Resolving as this is stylistic without a established pattern.